scene/resources/particles_material.cpp

```diff
@@ -659,8 +659,15 @@ void ParticlesMaterial::_update_shader() {
 	}
 	//scale by scale
 	code += "	float base_scale = tex_scale * mix(scale, 1.0, scale_random * scale_rand);\n";
+
 	// Prevent zero scale (which can cause rendering issues).
-	code += "	base_scale = sign(base_scale) * max(abs(base_scale), 0.000001);\n";
+	code += "	if (base_scale >= 0.0) {\n";
+	code += "		base_scale = max (base_scale, 0.000001);\n";
+	code += "	}\n";
+	code += "	else {\n";
```

Comment:
```suggestion
	code += "	} else {\n";
```
Following the Godot style guide all the way into shader land.